src/server/routes/flowerRoutes.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import { getAllFlowers, createFlower } from './controllers/flowerController.js';
+
+const router = express.Router();
```

Comment:
change name for each routr that you do to him export. 
cant be the same name for many vars